python/semantic_kernel/connectors/memory/astradb/astra_client.py

```diff
@@ -1,11 +1,20 @@
 import json
+from importlib.metadata import PackageNotFoundError, version
 from typing import Dict, List, Optional
 
 import aiohttp
 
 from semantic_kernel.connectors.memory.astradb.utils import AsyncSession
 from semantic_kernel.exceptions import ServiceResponseException
 
+SEMANTIC_KERNEL_VERSION: Optional[str]
+try:
+    SEMANTIC_KERNEL_VERSION = version("semantic_kernel")
```

Comment:
`semantic_kernel` should be `semantic-kernel` and I'm also wondering if you could reuse the APP_INFO in connectors/telemetry.py, could you have a look at that?

Reply:
Thank you for the tip! I updated the flow accordingly, much cleaner now :)